src/RenderedHelper.ts

```diff
@@ -1,16 +1,17 @@
 import * as vscode from 'vscode';
 import { CONSTANTS } from './constants';
 import { PdfPreview } from './PdfPreview';
-import { RenderTemplateStepResult, StitchResponse } from './types';
+import { HttpStepConfiguration, RenderTemplateStepResult, StitchResponse } from './types';
 
 export class RenderedHelper {
-
+    
     public static show(options: { filename: string; content: string; }): void {
         if (!options.content) { return; }
 
         const firstChar = options.content[0];
         let extension = '.txt';
-        if (options.content.startsWith('<!DOCTYPE html>') || options.content.startsWith('<html>')) { extension = '.html'; }
+        if (options.filename.endsWith(CONSTANTS.httpFileExtension)) { extension = ''; }
```

Comment:
Don't you want the extension to be `.http` here?

Reply:
Would have liked that but then needed to parse the request line and account for all scenarios of HTTP request/response